docs/extensions/createExtensions.md

```diff
@@ -0,0 +1,169 @@
+---
+sidebar_position: 11
+---
+
+# Creating Your Own Extension
+
+This section will help you develop custom extensions for Scaffold-ETH 2, from simple additions to more complex modifications.
+
+> Video Guide: For a visual walkthrough of the extension development process, check out our [YouTube tutorial](https://youtu.be/XQCv533XGZk?si=dlJH4zd4b99_6soW).
+
+## Extension Structure
+
+Before diving into the development process, let's understand the structure an extension should follow:
+
+```
+your-extension/
+├── extension/
+│   ├── packages/
+│   │   ├── hardhat/        # (optional) For Hardhat-specific additions
+│   │   ├── foundry/        # (optional) For Foundry-specific additions
+│   │   └── nextjs/
+│   │       ├── app/        # any new pages/files
+│   │       │   └── my-page
+│   │       │       ├── page.tsx
+│   │       │
+│   │       ├── ...         # any extra files/directories
+│   │       └── package.json  # Only include additional dependencies/scripts
+│   ├── package.json        # monorepo root package.json file
+│   └── README.md           # Instance README
+└── README.md               # Documentation for your extension
+```
+
+## Developing a Simple Extension
+
+For simple extensions, such as adding a new page or component, you can directly create the extension structure without going through the full development workflow. Here's how:
+
+1. Create the directory structure as shown above.
+2. Add your new page or component in the appropriate directory.
+3. If needed, create a `package.json` with any additional dependencies.
+4. Push your extension to GitHub.
+
+That's it! Your simple extension is ready to be used by others via:
+
+```shell
+npx create-eth@latest -e {your-github-username}/{extension-repo-name}:{branch-name} # branch-name is optional
```

Comment:
Reminder, in case we decide to delete "your" from extensions.md we'd need to delete it here also 🙏

Reply:
yes - removed your from all to make it more clear! agree!!